src/consumer/lanes/merge-lanes.ts

```diff
@@ -37,7 +38,15 @@ export async function mergeLanes({
   let otherLane: Lane | null;
   let remoteLane;
   let otherLaneName: string;
-  if (remoteName) {
+  const isDefaultLane = laneName === DEFAULT_LANE;
+
+  if (isDefaultLane) {
+    const allBitIdsOnDefaultLaneWithVersion = consumer.bitMap
+      .getAuthoredAndImportedBitIdsOfDefaultLane()
+      .filter((id) => id.hasVersion());
+    bitIds = allBitIdsOnDefaultLaneWithVersion.map((id) => id.changeVersion(id.version));
```

Comment:
@luvkapur , not sure what this line does. seems like it changes the version but to the same version it has already.

Reply:
@davidfirst, that's an oversight from my side while refactoring. Since, we already have the versions from the bitmap for all the components there is no need to change the version. Pushed the fix.